pkg/storage/etcd/etcd_helper.go

```diff
@@ -154,7 +155,6 @@ type etcdHelper struct {
 	// This depends on etcd's indexes being globally unique across all objects/types. This will
```

Comment:
that is a problem then.  a preexisting one that is; not one that this PR creates.